package.json

```diff
@@ -79,10 +79,10 @@
     "tsconfig.esm.json"
   ],
   "scripts": {
-    "typecheck": "tsc --noEmit",
+    "typecheck": "npx tsc --noEmit",
     "test": "NODE_OPTIONS=\"--max_old_space_size=4096 --experimental-vm-modules=true\"  dotenv -e .env.test.local -- npx jest --config='./jest.config.js' --maxWorkers=2 --watch --verbose",
     "test:ci": "NODE_OPTIONS=\"--max_old_space_size=4096 --experimental-vm-modules=true\"  dotenv -e .env.test.local -- npx jest --config='./jest.config.js' --maxWorkers=2",
-    "build": "rimraf dist && npm run build:esm && tsc && tsc-alias",
+    "build": "rimraf dist && npm run build:esm && npx tsc && npx tsc-alias",
```

Comment:
what's the purpose of this? why didn't you also change the build commands?

Reply:
I ran into an issue where my compiler was failing. Turns out I was using my global tsc which was on wrong version.

npx forces the use of correct package version (the one installed in project)

Reply:
Good spot, will also fix build:esm